cloud/user/user_test.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserInvite(t *testing.T) {
+	testUtil.InitTestConfig(testUtil.LocalPlatform)
+
+	expectedInvite := astro.UserInvite{
+		UserID:         "",
+		OrganizationID: "",
+		OauthInviteID:  "",
+		ExpiresAt:      "",
+	}
+	jsonResponse, err := json.Marshal(expectedInvite)
+	assert.NoError(t, err)
+	client := testUtil.NewTestClient(func(req *http.Request) *http.Response {
+		return &http.Response{
+			StatusCode: 200,
+			Body:       io.NopCloser(bytes.NewBuffer(jsonResponse)),
+			Header:     make(http.Header),
+		}
+	})
+	astroAPI := astro.NewAstroClient(client)
+
+	invite, err := CreateInvite("test", "test1", astroAPI)
+	assert.NoError(t, err)
+	// TODO this feels incorrect.
+	// Should we test if client.CreateUserInvite() was called with the correct input ?
+	// Both happy path and error cases are tested in astro client already
+	// TODO fix the assertion based on discussion
```

Comment:
We should rather rely on the mocks generated for the astro-client instead of creating a new mock HTTP client, that will allow us to check if the input to the method called for those mocks is correct or not.

Attaching an example for your reference: https://github.com/astronomer/astro-cli/blob/main/cloud/deployment/deployment_test.go#L22-L23

Reply:
Thanks and changed it.